manual/manual/cmds/unified-options.etex

```diff
@@ -393,8 +393,13 @@ libraries and options on the command line.
 
 \nat{%
 \item["-nodynlink"]
-Allow the compiler to use some optimizations that are valid only for code
-that is never dynlinked.
+Allow the compiler to use some optimizations that are valid only for
+code that is statically linked to produce a non-relocatable
+executable.  The generated code cannot be linked to produce a shared
+library nor a position-independent executable (PIE).  Many Linux
+systems produce PIEs by default, causing errors when linking code
```

Comment:
Minor nitpick: do we want to keep an up-to-date list of OSs with PIEs as default? For instance, openBSD seems to also produce PIE executable by default.

Reply:
> do we want to keep an up-to-date list of OSs with PIEs as default?

No we don't want, because we are unable to.  

However I can make the advice less specific: "Many operating systems produce PIEs by default..."

Reply:
This less specific wording seems more future-proof.

Reply:
Done.